21/day01/src/main.rs

```diff
@@ -0,0 +1,41 @@
+fn main() {
```

Comment:
Use for the commit the English language and adjust your own conventions.

Reply:
Do you approve of the commit messages? If so, please mark this thread as resolved.